app/models/appointment.rb

```diff
@@ -6,4 +6,22 @@ class Appointment < ApplicationRecord
   belongs_to :user
   validates_presence_of :start_time
   validates_presence_of :appointment_date
+
+
+  def self.todays_appointments
+    appointment_date = Time.new(Date.today.year, Date.today.month, Date.today.day, 0, 0, 0, "UTC")
+    logger.debug appointment_date
+    todays_appointments = []
+
+    appointments_for_today = Appointment.where(appointment_date: appointment_date)
+    logger.debug appointments_for_today
```

Comment:
I don't think it's a good practice to keep this debugger. Also, you can use byebug to debug stuff